docs/general/via-parity-signer.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Via Parity Signer
+sidebar_position: 2
+sidebar_label: Via Parity Signer
+---
+
+import googleURL from './img/google-play-badge.png';
+import appleURL from './img/app-store-badge.png';
+
+# Vara cold storage via Parity Signer
+
+Parity Signer allows you to turn any mobile device based on Android or iOS into a real cold walllet. Your private keys are kept secure offline at all times and transactions are signed in an air-gapped way via QR-codes. A huge advantage is the completely open and transparent source code of [the product](https://github.com/paritytech/parity-signer)
+
+Vara Network completely supports working with Parity Signer.
+
+## Installation
+
+There are several ways to get the Parity Signer
+
+Download from the application store for your device:
+
+<div align="left">
+  <a href="https://play.google.com/store/apps/details?id=io.parity.signer">
+    <img src={googleURL} alt="Play Market" />
+  </a>{' '}
+  <a href="https://itunes.apple.com/us/app/parity-signer/id1218174838">
+    <img src={appleURL} alt="Apple Store" />
+  </a>
+</div>
+
+Download directly from the github:
+
+https://github.com/paritytech/parity-signer/releases
+
+## Create your keys
+
+:::info
```

Comment:
Maybe some screenshots here with explanation how single seed phrase can be used with any amount of public addresses via salt?